src/Nethermind/Nethermind.Init/Steps/EthereumStepsManager.cs

```diff
@@ -88,6 +89,26 @@ private List<Task> CreateAndExecuteSteps(CancellationToken cancellationToken)
                 }
             }
 
+            // Remove absent optional steps
```

Comment:
Can we not remove them, but don't add them in the first place in the above loop?

Reply:
Yep, moved to the loop above

Reply:
removed ToArray